### ISSUE TITLE
Access Denied Problem Fix THis

### ISSUE THREAD

**Issue by Lennon2563** [noahcoolboy/funcaptcha#43]
![e24532f7030c5b4bd1eecdbab7fefbe2](https://github.com/noahcoolboy/funcaptcha/assets/137152805/7e4b6adb-dd7d-486c-a66b-21d69dba5b50)

code
```javascript
const blob = req.body.data.blob;
    const token = await funcaptcha.getToken({
        pkey: req.params.public_key,
        surl: "https://roblox-api.arkoselabs.com",
        data: {
            blob: blob 
        },
        headers: { // OPTIONAL
            "User-Agent": 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.0.0 Safari/537.36'
        },
        site: "https://www.roblox.com/login",

    })
    res.status(200).send(token);
```

**Comment by Lennon2563** [comment]
I think its user agent problem i hope you fix thank you

**Comment by noahcoolboy** [comment]
Fixed in 6926450186845caf3757977e6578fba10e4a4b74
Please update to v1.1.1
